forward_command_controller/include/forward_command_controller/forward_joint_group_command_controller.h

```diff
@@ -98,6 +103,7 @@ class ForwardJointGroupCommandController: public controller_interface::Controlle
         return false;
       }
     }
+    commands_buffer_.readFromRT()->resize(n_joints_, 0.0);
```

Comment:
Use [`readFromNonRT()`](https://github.com/ros-controls/realtime_tools/blob/indigo-devel/include/realtime_tools/realtime_buffer.h#L109) here. The implementation will make sure that it waits until it gets the shared resource. It's OK to wait for a resource during `init`.